src/test/resources/ide/language/rust/Constructs.rs

```diff
@@ -0,0 +1,23 @@
+
+pub struct <warning>Peerson</warning> {
+    <warning>naame</warning>: String,
+}
+
+impl Peerson {
+
```

Comment:
Cleanup here and everywhere unneded whitespaces

Reply:
Done